packages/zod-openapi/src/index.test-d.ts

```diff
@@ -240,7 +240,7 @@ describe('coerce', () => {
     type Actual = ExtractSchema<typeof routes>['/api/users/:id']['$get']['input']
     type Expected = {
       param: {
-        id: number
+        id: unknown
```

Comment:
The `input` type of `coerce` will be `unknown` in the v4:

https://zod.dev/v4/changelog#zcoerce-updates